lib/neo4j/railtie.rb

```diff
@@ -4,7 +4,7 @@
 require 'action_dispatch/railtie'
 
 module Neo4j
-  class Railtie < ::Rails::Railtie
+  class Railtie < Rails::Railtie
```

Comment:
I was checking edge api http://edgeapi.rubyonrails.org/classes/Rails/Railtie.html and they dont use ::Rails::Railtie, I'll check it with that.
I think what it works here is the require line about neo4j/railtie. I'll do more tests.